test/Context.spec.js

```diff
@@ -819,5 +819,19 @@ describe('Context', () => {
 
 		refs2.dispatchEvent(new Event('click'))
 		assert.html(container, `<button style="color: orange;">2</button><button>Colors!</button>`)
+	}),
+
+	it('should write and read propTypes of context providers properly', () => {
```

Comment:
We can add this to `PropTypes.spec.js` when we update context to support `propTypes`.

Reply:
Uuups, thanks for the info ... I've completely forgotten that unit test :-( ... is removed